src/core/server/deprecations/index.ts

```diff
@@ -6,10 +6,13 @@
  * Side Public License, v 1.
  */
 
-export type { GetDeprecationsContext, RegisterDeprecationsConfig } from './types';
-
 export type {
   DeprecationsServiceSetup,
+  GetDeprecationsContext,
+  RegisterDeprecationsConfig,
+} from '@kbn/core-deprecations-server';
+
+export type {
```

Comment:
We need to update https://github.com/elastic/kibana/issues/135961 with the remaining work to migrate the deprecations service.